CHANGELOG.md

```diff
@@ -2,6 +2,48 @@
 
 This contains only the most important and/or user-facing changes; for a full changelog, see the commit history.
 
+## [0.15.0](https://github.com/ably/ably-chat-js/tree/0.15.0) (TBD)
+
+### Breaking Changes
+
+This release contains significant API changes to improve consistency and type safety. Please see `UPGRADING.md` for full guidance on upgrading from version 0.14.x.
+
+- **Typing and Occupancy API**: Changed `current()` methods to properties for consistency with other SDK APIs. [#682](https://github.com/ably/ably-chat-js/pull/682)
+- **Message API Simplification**: Removed convenience comparison methods (`before()`, `after()`, `equal()`, etc.) from Message type. [#676](https://github.com/ably/ably-chat-js/pull/676)
+- **Presence Event Filtering**: Removed event filter parameter from presence subscription for API consistency. [#675](https://github.com/ably/ably-chat-js/pull/674)
+- **Message Serial Parameters**: Serial parameters in method signatures must now be strings instead of objects. [#673](https://github.com/ably/ably-chat-js/pull/673)
+- **Messages.delete Signature**: Updated signature to be consistent with `Messages.update`, using `OperationDetails` type. [#671](https://github.com/ably/ably-chat-js/pull/671)
```

Comment:
I probably wouldn't mention this - seeing as you can't "name" parameters whilst calling